react/fields/FieldLabel/FieldLabel.less

```diff
@@ -0,0 +1,11 @@
+@import (reference) "~seek-style-guide/theme";
+
+.label {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    font-weight: @sk-bold;
+  }
+}
+
+.secondaryLabel {
+  font-weight: 300;
```

Comment:
Shouldn't this either be an existing weight, or we should be creating a new variable for it?